### PR TITLE
Add limits for token size #258

### PR DESCRIPTION
**Pull request by pensan**
Tried to add scaling limits (#258).

Used `25px` as min size, and `1000px` as max size in the Token class. Please change the values if you find other values more appropriate.

Also added a clamp to the `Token Image Scale` slider that matches the HTML form input values.

### CODE REVIEW

**Comment by pensan** [comment]
Closes #258

File: Token.js [hunk 1]
```diff
@@ -135,6 +138,9 @@ class Token {
 	}
 
 	size(newsize) {
+		// Clamp token size to min/max token size
+		newsize = clamp(newsize, this.MIN_TOKEN_SIZE, this.MAX_TOKEN_SIZE);
```
Comment (vlaminck) [inline Token.js:142]: I think we should clamp the upper bound to `$("#scene_map").width()` as long as that exists

**Comment by bain2236** [comment]
Shouldn't the max of a token be the same size as the limits going on in your other branch? If they were both merged in you could have a token largeer than the map and not be moveable. Currently I can end up with something like this which feels strange as a max size. it might also be nice to show hte user when they've hit the max size for a token within the context menu token adjustments flyout. Although I think there's some changes in the big token PR that fixes some things in that menu
image

**Comment by pensan** [comment]
I updated the logic to better fit the needs.

The scaling is now adjusted when the token size changes. 
Allowing for a max token size of `$("#scene_map").width()`.

Scaling is then adjusted to the new token size.
A token of 'full map with' size will only be able to have a image scale of "1" or smaller.

If the token size is changed to a larger value, the scale is reset to "1".

The DOM inputs are also updated with new max values when token size changes.

**Comment by pensan** [comment]
The conflict should be resolved by using both revisions.
We need all 3 variables

**Comment by pensan** [comment]
Rebased and refactored to work with the new Token/TokenMenu changes.

Also fixes a bug in relation to https://github.com/cyruzzo/AboveVTT/issues/469 which occurred when a token size was changed after initial load.

**Pull request merged**